vavr/src/main/java/io/vavr/collection/Iterator.java

```diff
@@ -452,6 +452,18 @@ public Short getNext() {
         return io.vavr.collection.Collections.fill(n, s);
     }
 
+    /**
+     * Returns a Iterator containing {@code n} times the given {@code element}
+     *
+     * @param <T>     Component type of the Iterator
+     * @param n       The number of elements
+     * @param element The element
+     * @return An iterator on a sequence of {@code n} elements, where each element are given {@code element}.
```

Comment:
An iterator **of {\@code n} sequential elements,** where each element ~are~ **is the** given {\@code element}.